ui/pages/unlock-page/index.scss

```diff
@@ -1,63 +1,15 @@
 @use "design-system";
```

Comment:
Removing a lot of unneeded CSS now that we are using the Box component and style utility classnames